src/components/Pagination.jsx

```diff
@@ -42,7 +42,7 @@ function Pagination({
           // Do not remove the aria-label below, it is used for Hatchways automation.
           aria-label="Goto previous page"
           onClick={onPrevious}
-          disabled={false} // change this line to disable a button.
+          disabled={currentPage > 1 ? false : true} // change this line to disable a button.
```

Comment:
This logic can be refactored to be more concise as the left side of the expression already returns a boolean so the ternary operator is not required. This is also applicable to line 88 below.